cirq-core/cirq/transformers/gauge_compiling/sqrt_cz_gauge.py

```diff
@@ -14,16 +14,44 @@
 
 """A Gauge transformer for CZ**0.5 gate."""
```

Comment:
Please update w/r to the support of the CZ**-0.5 gate.

Reply:
done